Readme.md

```diff
@@ -22,6 +26,7 @@ UnicornWrangler.setup(
     check_every: 250 # requests
   },
   gc_after_request_time: 10, # seconds
+  finish_requests_before_stopping: true,
```

Comment:
I would rename this into `remap_term_into_quit`. Finishing a request before stopping is something that is by default as long as you follow Unicorn convention of sending QUIT. 

It's only on K8s and Heroku that the platform sends `TERM`. So if one is using this gem outside K8s and activate this options `finish_requests_before_stopping` and sends QUIT will actually obtain something different

Reply:
It's kind of "what effect does this have" vs "how is this implemented"

so either:
 - `finish_requests_before_stopping: true # maps TERM to QUIT for heroku or kubernetes`
 - `map_term_to_quit: true # finish requests before stopping`

I'm not expecting the implementation to change, so the second seems better ...